doozerlib/cli/images_streams.py

```diff
@@ -1237,8 +1260,6 @@ def check_if_upstream_image_exists(upstream_image):
                         # In one execution to date, get_pulls did not find the open PR and the code repeatedly hit this
                         # branch -- attempting to recreate the PR. Everything seems right, but the github api is not
                         # returning it. So catch the error and try to move on.
-                        pr_url = f'UnknownPR-{fork_branch_head}'
-                        pr_links[dgk] = pr_url
```

Comment:
since we will use pr_links to do jira connection in the following step, such exception info is not useful in pr_links list.